pkg/prometheus/operator.go

```diff
@@ -122,6 +122,27 @@ func ValidateRemoteWriteSpec(spec monitoringv1.RemoteWriteSpec) error {
 	return nil
 }
 
+func ValidateAlertmanagerEndpoints(am monitoringv1.AlertmanagerEndpoints) error {
```

Comment:
I think this function may fits better under the AM [validation.go](https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/alertmanager/validation/validation.go)

Reply:
Hi @nicolastakashi, thanks for the review! I've checked the file, but it appears that the file is more related to the Alertmanager's configuration. What I'm adding here is the Prometheus' AlertmanagerEndpoints configuration, and I noticed there is another 'ValidateRemoteWriteSpec' function in this 'prometheus/operator.go' file as well. In my opinion, it makes more sense to place this function in the 'prometheus' folder? 😄